src/Camera.cpp

```diff
@@ -1,29 +1,58 @@
-
 #include "Camera.h"
 
 #include <GLFW/glfw3.h>
 #include <glm/gtc/matrix_transform.hpp>
 #include <glm/mat4x4.hpp>
-
-
+#include <glm/gtx/transform.hpp>
+#include <glm/glm.hpp>
 
 Camera::Camera()
 	: ProjectionMatrix(glm::mat4(1.0))
 	, ViewMatrix(glm::mat4(1.0))
 {
 	// Initial position
-	glm::vec3 position = glm::vec3(4,3,3);
+	pos = glm::vec3(4,3,3);
+	forward = glm::vec3(0,0,0);
+	up = glm::vec3(0,1,0);
+
 	// Initial Field of View
 	float initialFoV = 45.0f;
 
 	ViewMatrix = glm::lookAt(
```

Comment:
Left bracket should move to next line to keep consistent with rest of the code.

Reply:
d